image.scala.yaml

```diff
@@ -1,7 +1,7 @@
 name: "radanalytics-scala-spark"
 version: 1.0
 release: dev
-from: "radanalyticsio/openshift-spark:2.2-latest"
+from: "rimolive/scala-211-centos7:latest"
```

Comment:
Probably need to host this image under radanalyticsio to avoid using anyone's personal bits